e2e-tests/custom-login/specs/custom-login-flow-spec.js

```diff
@@ -13,15 +13,18 @@
 'use strict';
 
 const LoginHomePage = require('../../page-objects/shared/login-home-page');
-const CustomSignInPage = require('../../page-objects/custom-signin-page');
+let OktaSignInPage = require('../../page-objects/okta-signin-page');
+if (process.env.ORG_OIE_ENABLED) {
+  OktaSignInPage = require('../../page-objects/okta-oie-signin-page');
+}
 const AuthenticatedHomePage = require('../../page-objects/shared/authenticated-home-page');
 const ProfilePage = require('../../page-objects/shared/profile-page');
 const MessagesPage = require('../../page-objects/messages-page');
 const url = require('url');
 
 describe('Custom Login Flow', () => {
   const loginHomePage = new LoginHomePage();
-  const customSignInPage = new CustomSignInPage();
+  const customSignInPage = new OktaSignInPage();
```

Comment:
`CustomSignInPage` class was exactly same as `OktaSignInPage`. Removed it, so that we can use the same `OktaSignInPage` object in both specs.